ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/ThreadNet/Network.hs

```diff
@@ -1012,12 +1024,13 @@ directedEdgeInner edgeStatusVar
       [ miniProtocol
           (wrapMPEE MPEEChainSyncClient naChainSyncClient)
           naChainSyncServer
+      -- Don't swallow exceptions thrown by these, let them fail the test
```

Comment:
I agree with this, and it's overdue. we need to trim down the set of non-fatal exceptions. I suspect that `ForkTooDeep` and `IntersectionNotFound` would suffice for the current tests. (For context: the testing plan I'm currently devising would avoid even those: any exception would be fatal.)